src/Main.cpp

```diff
@@ -181,23 +181,21 @@ int main(int argc, char** argv)
 	//       Hint: Use selectQueueFamilyIndex, but complete its implementation before!
 	uint32_t selected_queue_family_index = std::numeric_limits<uint32_t>::max();
 
-	// Sanity check:
-	{
-		uint32_t queue_family_count = 0;
-		vkGetPhysicalDeviceQueueFamilyProperties(vk_physical_device, &queue_family_count, nullptr);
-		if (selected_queue_family_index > queue_family_count) {
-			VKL_EXIT_WITH_ERROR("Invalid queue family index selected.");
-		}
-		VKL_LOG("Task 1.5 done.");
+	// Sanity check if we have selected a valid queue family index:
+	uint32_t queue_family_count = 0;
+	vkGetPhysicalDeviceQueueFamilyProperties(vk_physical_device, &queue_family_count, nullptr);
+	if (selected_queue_family_index > queue_family_count) {
```

Comment:
Shouldn't this be 
```c++
if (selected_queue_family_index >= queue_family_count) {
```
queue family indices should start with 0

Reply:
uh oh, yeah it should 🙈 good catch!

Reply:
Mhm, to be completely correct we would also need to check if the query family supports graphics/surface otherwise it could say task 1.5 done while I will not work later on

Reply:
But doing this without giving to much away could be hard

Reply:
Nah, just wanted to have a quick check that points out if the user has not properly assigned a queue family index, i.e., so that the program fails at the right location.